### PR TITLE
NO-JIRA Adding tests

### PR DESCRIPTION
**Pull request by andytaylor**

### CODE REVIEW

**Comment by mtaylor** [comment]
@andytaylor Could you ammend the commit message to something more descriptive please.

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/AdvisoryOpenWireTest.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+public class AdvisoryOpenWireTest extends BasicOpenWireTest {
+
+   @Override
+   @Before
+   public void setUp() throws Exception {
+      //this system property is used to construct the executor in
+      //org.apache.activemq.transport.AbstractInactivityMonitor.createExecutor()
+      //and affects the pool's shutdown time. (default is 30 sec)
+      //set it to 2 to make tests shutdown quicker.
+      System.setProperty("org.apache.activemq.transport.AbstractInactivityMonitor.keepAliveTime", "2");
```
Comment (clebertsuconic) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/AdvisoryOpenWireTest.java]: This kind of thing is a potential source of issues on another tests...

Can you cleanup the property on a tearDown please?

Comment (clebertsuconic) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/AdvisoryOpenWireTest.java]: Also +1 on @mtaylor ... a better description of the test you added would be great.

**Comment by andytaylor** [comment]
I just moved these tests from another class and added slightly different assertion.

**Comment by clebertsuconic** [comment]
@andytaylor ok.. the issue is just duplicated now ...

I can take care of the other property. What would be a better commit description here?

"NO-JIRA Adding Advisory OpenWire tests"

???


I can amend this and take care of the systemproperty myself. if you say it's ok the commit name.

**Comment by andytaylor** [comment]
yeah thats fine, cheers

**Pull request closed without merging**